src/lib/packager/clients/pkg_finish.rb

```diff
@@ -284,15 +285,24 @@ def sync_target_sources
       Pkg.TargetInitialize(Installation.destdir)
     end
 
-    # Set libzypp configuration to install the minimal amount of packages
+    # Set package manager configuration to install the minimal amount of packages
     #
     # @see Yast::Packager::CFA::ZyppConf#set_minimalistic!
-    def set_minimalistic_libzypp_conf
-      log.info("Setting libzypp configuration as minimalistic")
-      config = Packager::CFA::ZyppConf.new
-      config.load
-      config.set_minimalistic!
-      config.save
+    def set_minimalistic_pkg_conf
+      ["dnf", "libzypp"].each do |pkg|
```

Comment:
I do not like these artificial names , what about iterating over the class names instead?

```ruby
[ Packager::CFA::ZyppConf, Packager::CFA::DnfConf].each do |conf_class|
  next unless File.file?(File.join(Installation.destdir, conf_class.const_get(:PATH))
  config = conf_class.new
  config.load
  config.set_minimalistic!
  config.save
end
```

Reply:
Names are now only used for logging, that should work

Reply:
`conf_class::PATH` should also work (instead of that `conf_class.const_get`).